modules/graceful-shutdown.adoc

```diff
@@ -33,110 +33,56 @@ If your cluster fails to recover, follow the steps to restore to a previous clus
 
 .Procedure
 
-. If you plan to shut down the cluster for an extended period of time, determine the date that cluster certificates expire.
-+
-You must restart the cluster prior to the date that certificates expire. As the cluster restarts, the process might require you to manually approve the pending certificate signing requests (CSRs) to recover kubelet certificates.
-
-.. Check the expiration date for the `kube-apiserver-to-kubelet-signer` CA certificate:
+. If you are shutting the cluster down for an extended period, determine the date on which certificates expire.
 +
 [source,terminal]
 ----
-$ oc -n openshift-kube-apiserver-operator get secret kube-apiserver-to-kubelet-signer -o jsonpath='{.metadata.annotations.auth\.openshift\.io/certificate-not-after}{"\n"}'
+$ oc -n openshift-kube-apiserver-operator get secret kube-apiserver-to-kubelet-signer -o jsonpath='{.metadata.annotations.auth\.openshift\.io/certificate-not-after}'
 ----
 +
 .Example output
-[source,terminal]
-----
-2023-08-05T14:37:50Z
-----
-
-.. Check the expiration date for the kubelet certificates:
-
-... Start a debug session for a control plane node by running the following command:
-+
-[source,terminal]
 ----
-$ oc debug node/<node_name>
+2022-08-05T14:37:50Zuser@user:~ $ <1>
 ----
+<1> To ensure that the cluster can restart gracefully, plan to restart it on or before the specified date. As the cluster restarts, the process might require you to manually approve the pending certificate signing requests (CSRs) to recover kubelet certificates.
 
-... Change your root directory to `/host` by running the following command:
+. Mark the control plane nodes in the cluster as unschedulable. You can do this from your cloud provider's web console, or run the following loop:
```

Comment:
Hi, @nalhadef 
It seems keep not update. It should be: Mark all the nodes in the cluster as unschedulable.